### ISSUE TITLE
clang-format

### ISSUE THREAD

**Issue by tiagomacarios** [include-what-you-use/include-what-you-use#502]
Did anyone ever had a look at running clang-format after iwyu has run?

**Comment by kimgr** [comment]
You mean let IWYU format its edits? Not really, but it sounds easier to lean on clang-format's toolchain (e.g. clang-format-diff or git-clang-format) after IWYU has done its work and not commingle the two.

**Comment by tiagomacarios** [comment]
yes. some of the IWYU edits do not follow clang-format rules (AFAIR it was related to comment alignment).

So right now I have to IWYU and then clang-format. It would be nice if IWYU followed the clang-format rules, or if it called clang-format after done.

The rational is that it should be self contained and not need a clang-format pass afterwards.

**Comment by kimgr** [comment]
I'd rather keep them separate, but I'll leave this as an enhancement if someone wants to work on it.
